repos/spack_repo/builtin/packages/hipblaslt/package.py

```diff
@@ -122,9 +130,11 @@ def setup_build_environment(self, env: EnvironmentModifications) -> None:
                 "ROCM_AGENT_ENUMERATOR_PATH",
                 f"{self.spec['rocminfo'].prefix}/bin/rocm_agent_enumerator",
             )
+        if self.spec.satisfies("@7.0:"):
+            env.set("ROCM_LD_PATH", f"{self.spec['llvm-amdgpu'].prefix}/bin/ld.lld")
```

Comment:
I looked at https://github.com/ROCm/hipBLASLt/blob/rocm-7.0.0/tensilelite/Tensile/Toolchain/Validators.py#L118 , can you add this too for 7.0.0: ?
  env.set("C", f"{self.spec['llvm-amdgpu'].prefix}/bin/amdclang")